doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,164 @@
+SciPy Code of Conduct
+=====================
+
+
+Introduction
+------------
+
+This code of conduct applies to all spaces managed by the SciPy project,
+including all public and private mailing lists, issue trackers, wikis, blogs,
+Twitter, and any other communication channel used by our community.  The SciPy
+project does not organise in-person events, however we expect events relevant
+to our community to have a code of conduct similar in spirit to this one.
+
+We expect this code of conduct to be honored by everyone who participates in
+the SciPy community formally or informally, or claims any affiliation with the
+project, in any project-related activities and especially when representing the
+project, in any role.
+
+This code is not exhaustive or complete. It serves to distill our common
+understanding of a collaborative, shared environment and goals. We expect it to
+be followed in spirit as much as in the letter, so that it can enrich all of us
+and the technical community in which we participate.
+
+
+Specific Guidelines
+-------------------
+
+We strive to:
+
+1. Be open. We invite anyone to participate in our community. We preferably use
+   public methods of communication for project-related messages, unless
+   discussing something sensitive. This applies to messages for help or
+   project-related support, too; not only is a public support request much more
+   likely to result in an answer to a question, it also makes sure that any
+   inadvertent mistakes made by people answering will be more easily detected
+   and corrected.
+
+2. Be empathetic, welcoming, friendly, and patient. We work together to resolve
+   conflict, assume good intentions, and do our best to act in an empathetic
+   fashion. We may all experience some frustration from time to time, but we do
+   not allow frustration to turn into a personal attack. A community where
+   people feel uncomfortable or threatened is not a productive one. We should
```

Comment:
I'm not confident the last sentence here adds much.  Consider removing?  If you feel that the word "respect" should be used, include in the first sentence.